core/scripts/Voting.js

```diff
@@ -107,34 +185,36 @@ function fetchConstantPrice(request, config) {
   return web3.utils.toWei(config.value);
 }
 
-async function fetchIntrinioEquitiesPrice(request, config) {
-  const requestMoment = moment.unix(request.time);
+function getIntrinioTimeArguments(time) {
+  const requestMoment = moment.unix(time);
   const startDate = requestMoment.format("YYYY-MM-DD");
   const startTime = requestMoment.format("HH:mm:ss");
 
   // How to determine this time window? Picked 10000 seconds arbitrarily.
   const timeWindowSeconds = 10000;
-  const endMoment = moment.unix(Number(request.time) + timeWindowSeconds);
+  const endMoment = moment.unix(Number(time) + timeWindowSeconds);
```

Comment:
Warning: this comment may be an unnecessary amount of thinking for the current state of the AVS.

My rough understanding of the way Intrinio's apis generally work is that the 0th element in the list is the most *recent* time that satisfies the constraints. For instance, if you specify no constraints, the 0th element will be the most recent tick available. If you specify a window that starts at the request time and goes into the future, one would expect the 0th element to usually be a time past the request time even if a closer one is available in the specified window.

There aren't any perfect solutions to this problem, but do you think it would make sense to set the end time to the request time and not set a start time? I think this approach has two advantages:
1. If there is a tick right on or just before the requested time, that gets returned. You still wouldn't get a tick that is just *after* the request time, however.
2. It's somewhat intuitive to assume that with time series data, if there is no sample at the point you requested, you would want to use the closest point *before* the request rather than after since the sample after contains information that was not yet known at the request time while the one before doesn't. Put another way, the sample before is what should've been returned as the realtime or prevailing price at the request time.

I think my reasoning above could be flawed, so I think we should discuss in more detail before making any changes.